### ISSUE TITLE
Proper way to prevent ACPCore.start() more than once

### ISSUE THREAD

**Issue by booker-dragon** [adobe/react-native-acpcore#40]
### Expected Behaviour
The app should not do `ACPCore.start()` again after reload by getting a status value from native code

### Actual Behaviour
No API to get the status of SDK, whether it is properly started or not

### Reproduce Scenario (including but not limited to)
Have SDK initialization on app start and some callback method after we await for the result from `ACPCore.start()`

#### Steps to Reproduce
Load the app and SDK should start properly, do a reload (CMD + R / R + R)

#### Platform and Version
React Native

#### Sample Code that illustrates the problem
```
async componentDidMount() {
    await ACPCore.start()
    // below line will never execute after a reload as ACPCore.start() never resolve when it is called more than once
    doSomethingHere()
}
```

**Comment by nporter-adbe** [comment]
Hi,

This is expected behavior when you try to initialize the SDK more than once and should not occur in the real world. You should see a log with a message similar to: `Can't execute Core start action more than once.`.

Please follow the documentation suggestion of initializing the SDK via native code to avoid this issue among others. You _could_ also remove the `await` keyword to get around this during development as this shouldn't ever happen outside of development, and add the `await` back in for production.

https://github.com/adobe/react-native-acpcore#initializing-the-sdk

**Comment by nporter-adbe** [comment]
We're also going to work on a React Native fix so this promise is always resolved.

**Comment by nporter-adbe** [comment]
@booker-dragon The fix has been released in `v1.1.1`, please upgrade.

**Comment by booker-dragon** [comment]
Appreciate!
